xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererDMA.h

```diff
@@ -19,7 +19,7 @@ namespace RETRO
 class CRendererFactoryDMA : public IRendererFactory
 {
 public:
-  ~CRendererFactoryDMA() override = default;
+  virtual ~CRendererFactoryDMA() override = default;
```

Comment:
`virtual` shouldn't be required, right?

Reply:
right, I'm just not sure why the build is failing on jenkins

```
08:58:38 [100%] Linking CXX executable kodi-wayland
08:58:46 /home/jenkins/workspace/LINUX-64-Wayland/xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererDMA.h:19: error: undefined reference to 'vtable for KODI::RETRO::CRendererFactoryDMA'
08:58:46 /usr/bin/ld.gold: the vtable symbol may be undefined because the class is missing its key function
```

Reply:
Try sticking the `= default` destructor bodies in .cpp files. Start with CRendererFactoryDMA.

Reply:
I think I figured it out :man_facepalming: jenkinsbuilds wayland-gl and I had it set to only build for wayland-gles